client/src/components/Community/Community.css

```diff
@@ -0,0 +1,26 @@
+.community-component
+{
+     margin-top:100px;
+   
+}
+
+
+.Society-cards .heading
+{
+  text-align: center;
+  color: #ffffff;
+  font-style: italic;
+  font-weight: 200;
```

Comment:
You may have the same tab size for consistency throughout the code.

Reply:
I have fixed